src/server/api/controllers/users-controller.js

```diff
@@ -2,12 +2,24 @@ const knex = require('../../config/db');
 
 const getUserById = async (id) => {
   try {
-    const user = await knex('users').select('user_name', 'id').where({ id });
+    const user = await knex('users')
+      .select('user_name', 'id')
+      .where({ id });
```

Comment:
Again Tariq this is not related to your PR!

Reply:
,i just took changes from develop  then merge in feature branch,  i don't know  how is come!